libs/designsystem/src/lib/scss/_global-styles.scss

```diff
@@ -2,6 +2,7 @@
 @import './base/ionic';
 @import './base/typography';
 @import './base/line-clamp';
+@import 'print';
```

Comment:
Should we consider not exposing this through `global-styles` but instead have consumer projects that want to use print styles import them directly? Such that if they want to define their own they do not have to combat our styles. 

(Might be easier to discuss this IRL)